File: app/src/main/java/com/app/memoeslink/adivinador/AboutActivity.java
package com.app.memoeslink.adivinador;

import android.os.Bundle;
import android.text.method.LinkMovementMethod;
import android.widget.Button;
import android.widget.TextView;

public class AboutActivity extends CommonActivity {
    private TextView content;
    private Button button;

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);
        content = findViewById(R.id.about_content);
        content.setClickable(true);
        content.setText(Methods.fromHtml(getString(R.string.about)));
        content.setMovementMethod(LinkMovementMethod.getInstance());
        button = findViewById(R.id.about_back_button);

        //Set listeners
        button.setOnClickListener(view -> finish());
    }
}

File: app/src/main/java/com/app/memoeslink/adivinador/AboutActivity.kt
package com.app.memoeslink.adivinador

import android.os.Bundle
import android.text.method.LinkMovementMethod
import android.widget.Button
import android.widget.TextView

class AboutActivity : CommonActivity() {
    private var tvContent: TextView? = null
    private var btBack: Button? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_about)
        tvContent = findViewById(R.id.about_content)
        tvContent?.isClickable = true
        tvContent?.text = TextFormatter.fromHtml(getString(R.string.about))
        tvContent?.movementMethod = LinkMovementMethod.getInstance()
        btBack = findViewById(R.id.about_back_button)

        //Set listeners
        btBack?.setOnClickListener { finish() }
    }
}